src/ui/pages/ManageWallets/WalletAccount/WalletAccount.tsx

```diff
@@ -301,6 +302,11 @@ export function WalletAccount() {
               },
             ]}
           />
+          {!removeAddressMutation.isLoading && removeAddressMutation.error ? (
```

Comment:
Can this be changed to
```suggestion
          {removeAddressMutation.isError ? (
```